assets/src/components/reader-theme-carousel/index.js

```diff
@@ -149,22 +149,23 @@ export function ReaderThemeCarousel() {
 		<div className="reader-theme-selection">
 			<p>
 				{
-					// @todo Probably improve this text.
-					__( 'Select the theme template for mobile visitors', 'amp' )
+					__( 'Choose the theme to be used for AMP pages. This theme will normally be exclusively shown to mobile visitors when using Reader template mode.', 'amp' )
```

Comment:
> ...when using Reader template mode.

This sounds a bit repetitive, given that the user would have already selected the Reader template mode to access this submenu. How about:

```suggestion
					__( 'Choose the theme to be used for AMP pages. This theme will normally be exclusively shown to mobile visitors.', 'amp' )
```